MVCDemos/Views/ContextMenu/Scrolling.cshtml

```diff
@@ -0,0 +1,16 @@
+<div class="target-area">
+    Right click here to show the context menu
+</div>
+
+@(Html.DevExtreme().ContextMenu()
+    .Width(200)
+    .Target(".target-area")
+    .OnItemClick("contextMenu_ItemClick")
+    .DataSource(d => d.Mvc().LoadAction("GetScrollingProducts"))
+)
+
+<script>function contextMenu_ItemClick(e) {
+        if(!e.itemData.Items) {
+            DevExpress.ui.notify("The \"" + e.itemData.text + "\" item was clicked", "success", 1500);
```

Comment:
Using of string literals is more readable I think

Reply:
[da27f48](https://github.com/DevExpress/devextreme-demos/pull/3203/commits/da27f489909f7ec7cf3495811f9afbd46857a1c9)